src/oc/web/api.cljs

```diff
@@ -824,4 +826,15 @@
           (dispatcher/dispatch! [:comment-add/finish {:success success
                                                       :error (when-not success body)
                                                       :body (if (not (empty? body)) (json->cljs body) nil)
-                                                      :entry-uuid entry-uuid}]))))))
\ No newline at end of file
+                                                      :entry-uuid entry-uuid}]))))))
+
+(defn toggle-reaction
+  [topic-slug entry-uuid reaction-data]
+  (when (and topic-slug entry-uuid)
+    (let [reaction-link (or (utils/link-for (:links reaction-data) "react" "PUT")
```

Comment:
Might be good to have a version of `link-for` that's either `(utils/link-for links "react")` or `(utils/link-for links ["PUT" "DETELE"])`